src/main/java/com/beecoders/ras/service/dish/IngredientService.java

```diff
@@ -0,0 +1,24 @@
+package com.beecoders.ras.service.dish;
+
+import com.beecoders.ras.model.mapper.IngredientMapper;
+import com.beecoders.ras.model.response.dish.IngredientInfo;
+import com.beecoders.ras.repository.IngredientRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class IngredientService {
+    private final IngredientRepository ingredientRepository;
+    private final IngredientMapper ingredientMapper;
+
+    public void saveAll(List<IngredientInfo> ingredients) {
+        if (!ingredients.isEmpty())
```

Comment:
[_Not need to change_] It is better way to use libs from `apache` like `CollectionUtils` or `StringUtils` for validation because they provide null-safe methods.